tests/appsec/test_reports.py

```diff
@@ -24,7 +24,9 @@
 @released(golang={"gin": "1.37.0", "echo": "1.36.0", "*": "1.34.0"})
 @bug(library="python@1.1.0", reason="a PR was not included in the release")
 @missing_feature(weblog_variant="akka-http", reason="No AppSec support")
-@missing_feature(weblog_variant="spring-boot-payara", reason="No AppSec support")
+@missing_feature(
```

Comment:
Can you remove the java declaration on line 23 and use this : 

```suggestion
@released(java={"*":"0.92.0", "spring-boot-payara":"1.6.0"})
```

(or better, use manifest file)

Reply:
Done. If you don't mind, I'd prefer to migrate existing tests to the manifest in separate PRs, as these moves are relevant to more languages.